--- a/src/wfuzz/plugins/scripts/screenshot.py
+++ b/src/wfuzz/plugins/scripts/screenshot.py
@@ -3,7 +3,7 @@
 
 import subprocess
 import tempfile
-import pipes
+import shlex
 import os
 import re
 
@@ -42,7 +42,7 @@
         subprocess.call(
             [
                 "cutycapt",
-                "--url=%s" % pipes.quote(fuzzresult.url),
+                "--url=%s" % shlex.quote(fuzzresult.url),
                 "--out=%s" % filename,
                 "--insecure",
                 "--print-backgrounds=on",
